makefile

```diff
@@ -371,6 +371,14 @@ ifndef NOASM
 endif
 endif
 
+ifeq ($(findstring arm,$(UNAME)),arm)
+ARCHITECTURE :=
+endif
+
+ifeq ($(findstring aarch64,$(UNAME)),aarch64)
+ARCHITECTURE :=
+endif
+
```

Comment:
Why not put it in the other place where it looks for `arm` and `aarch64`?  Also, are the really no multilib AArch64 systems?

Reply:
I now put it together with where FORCE_C_DRC_BACKEND is set. Further consolidation (e.g. with line 172) is not possible without major refactoring because ARCHITECTURE is set to _x86 or _x64 based on PTR64 in line 340.
As to whether there is no multilib - I don't think so, gcc does not support -m32 or -m64 on arm and aarch64, respectively.